src/pages/SwapNew/components/CurrencyItem/CurrencyAmountWorth.tsx

```diff
@@ -0,0 +1,27 @@
+import { CurrencyAmount, Percent } from '@swapr/sdk'
+
+import styled from 'styled-components'
+
+import { TEXT_COLOR_SECONDARY } from '../../constants'
+import { FontFamily } from '../../constants'
+
+type CurrencyAmountWorthProps = {
+  fiatValue?: CurrencyAmount | null
+  priceImpact?: Percent
+  isFallback?: boolean
+}
+
+export function CurrencyAmountWorth({ fiatValue, priceImpact, isFallback }: CurrencyAmountWorthProps) {
```

Comment:
`priceImpact` is declared but never used

Reply:
I'll take a look. 👀